.agents/skills/testing-validation/SKILL.md

```
@@ -1,52 +1,26 @@
 ---
 name: testing-validation
-description: "Validate the chaotic_semantic_memory crate: compile, test, lint, LOC caps, property-based testing, fuzzing, and benchmarks. Use when asked to validate, check, or verify the build."
+description: "Validate the chaotic_semantic_memory crate: compile, test, lint, LOC caps, and benchmarks. Use when asked to validate, check, or verify the build."
 ---
 
-# Testing & Validation
-
-Comprehensive validation including property-based testing, fuzzing, and edge case coverage.
+# Testing Validation
 
 ## Quick Validation
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)